### PR TITLE
Dark theme button enabled

### PR DESCRIPTION
**Pull request by ChampGupta**

### CODE REVIEW

File: client/src/components/ThemeToggle.jsx [hunk 1]
```diff
@@ -1,50 +1,109 @@
...
+//               <path d="M21 12.79A9 9 0 1 1 11.21 3 7 7 0 0 0 21 12.79z"></path>
+//             </svg>
+//           </label>
+//   );
+// };
 
+// export default ThemeToggle;
+
+
+import * as React from 'react';
```
Comment (ChampGupta) [inline client/src/components/ThemeToggle.jsx:54]: Delete this if imported above

**Comment by ChampGupta** [comment]
I there is any issue remaining or any more improvement required please specify and I will fix it again.

**Comment by abhishekraoas** [comment]
> If there is any issue remaining or any more improvement required please specify and I will fix it again.

There is nothing happened after clicking the button. fix its not working

**Comment by ChampGupta** [comment]
@abhishekraoas please give me a few more time I am again working on this issue and will try to fix it by tomorrow

**Pull request closed without merging**